src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,375 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import {
+    CloudServiceProvider,
+    CreateRequest,
+    ServiceDetailVo,
+    ServicesAvailableService,
+    ServiceService,
+    ServiceVo,
+    UserAvailableServiceVo,
+} from '../../../../xpanse-api/generated';
+import { SelectDestination } from './SelectDestination';
+import { ShowDeploy } from './ShowDeploy';
+import { Button, Popconfirm, Space, Steps } from 'antd';
+import { DestroyService } from './DestroyService';
+import { OrderSubmitProps } from '../OrderSubmit';
+import { deployTimeout, destroyTimeout, usernameKey, waitServicePeriod } from '../../../utils/constants';
+import OrderSubmitResult from '../OrderSubmitResult';
+import OrderSubmitFailed from '../OrderSubmitFailed';
+import { getCreateRequest, getDeployParams, OperationType } from '../formElements/CommonTypes';
+import { ProcessingStatus } from '../ProcessingStatus';
+import { ExportServiceData } from './ExportServiceData';
+import { ImportServiceData } from './ImportServiceData';
+
+export const MigratingServices = ({
+    record,
```

Comment:
Better variable name instead of `record`

Reply:
fixed.